### DISCUSSION TITLE
CodeQL supports Ruby but not LGTM.com

### DISCUSSION THREAD

**Discussion by noraj** [community/community#24628] [Apps, API and Webhooks]
<p>If <a href="http://LGTM.com" rel="noopener nofollow ugc">LGTM.com</a> is based on CodeQL, why does it support all languages from CodeQL but Ruby?</p>
<ul>
<li><a href="https://lgtm.com/help/lgtm/getting-started" class="inline-onebox" rel="noopener nofollow ugc">LGTM</a></li>
<li><a href="https://codeql.github.com/docs/codeql-overview/supported-languages-and-frameworks/" class="inline-onebox" rel="noopener nofollow ugc">Supported languages and frameworks — CodeQL</a></li>
</ul>
<p>Or the LGTM documentation is outdated.</p>

**Comment by aibaars** [comment]
<p>There are Ruby projects available on <a href="http://LGTM.com" rel="noopener nofollow ugc">LGTM.com</a>; <a href="https://lgtm.com/search?q=language%3Aruby" class="inline-onebox" rel="noopener nofollow ugc">LGTM</a> . However, Ruby analysis is still beta.</p>

**Comment by sj** [comment, marked as answer]
<p>Thanks for dropping by, <a class="mention" href="/noraj">@noraj</a>! Just to add to <a class="mention" href="/aibaars">@aibaars</a>’s answer here: Ruby analysis is indeed in beta and therefore only available on GitHub code scanning for our beta testers to try out. Have you considered using GitHub code scanning to analyse your codebase? It’s based on the same CodeQL engine, and generally our latest and greatest features first appear there!</p>
